cypress.config.js

```diff
@@ -1,11 +1,26 @@
+const fetch = require('node-fetch');
 const { defineConfig } = require('cypress');
 
+const baseUrl = 'http://localhost:5678';
+
 module.exports = defineConfig({
 	e2e: {
-		baseUrl: 'http://localhost:5678',
+		baseUrl,
 		video: false,
 		screenshotOnRunFailure: true,
 		experimentalSessionAndOrigin: true,
 		experimentalInteractiveRunEvents: true,
+
+		setupNodeEvents(on) {
+			on('task', {
+				'db:reset': () => fetch(baseUrl + '/e2e/db/reset', { method: 'POST' }),
```

Comment:
Should these use `this.restEndpoint` to match most others?

Reply:
I'd like to keep all e2e related routes under `/e2e`, and skip as many of the middlewares on that route as possible.